openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -317,7 +324,29 @@ def list(self, request):
             search_usernames = usernames.strip(',').split(',')
         elif user_email:
             if is_email_retired(user_email):
-                return Response({'error_msg': RETIRED_EMAIL_MSG}, status=status.HTTP_404_NOT_FOUND)
+                cool_off_days = 14
```

Comment:
make this a settings variable instead of creating it within this block.

Reply:
Where should I move these? lms/env or cms/env? I can't see a settings path in openedx.

Reply:
@DawoudSheraz let me know about this too.

Reply:
LMS, because LMS is the source of truth for user information.